### PR TITLE
Added more infomation on Index-Mathing Materials for purchase.

### PR DESCRIPTION
**Pull request by sebyon**
I added a small table in the hardware section with some Index-Matching Material to choose from. Added a comment addressing the MICROFC-60035-SMT-TR and MICROFC-60035-SMT-TR1 difference (tape and reel purchase volume) explaining the price difference. 

Added some notes in the MD for future updates.

### CODE REVIEW

File: hardware/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This project utilizes a silicon photomultiplier (short SiPM) which is way smalle
 The following 6 mm SiPMs are recommended for this project:
 
 * **MICROFC-60035-SMT-TR**
-* MICROFC-60035-SMT-TR1 (the exact same as above, but much more expensive for whatever reason)
+* MICROFC-60035-SMT-TR1 (the exact same as above, however will be more expensive to purchase as sold in smaller batches to suppliers)
```
Comment (Phoenix1747) [inline hardware/README.md:10]: Can you explain this to me? Both on Mouser and Digikey you can buy individual SiPMs and they come in the same packaging in my experience.

File: hardware/README.md [hunk 2]
```diff
@@ -52,11 +52,11 @@ As you can see it is pretty easy to set the required voltage and threshold. **As
...
+The finished [MicroFC-](https://github.com/OpenGammaProject/MicroFC-SiPM-Carrier-Board) and [AFBR-](https://github.com/OpenGammaProject/AFBR-SiPM-Carrier-Board) SiPM carrier boards are there to allow for easier packaging with the scintillator as well as to be reusable for different detectors as that's by far the single most expensive part and you'll want to use it as long as possible.
 
-It's very important to get the SiPM/scintillator assembly light-tight. Otherwise you'll either run into problems with lower energies where noise dominates or outright not measure anything at all, because the sensor is saturating.
+### Scintillator Cyrstals
 
-**==> More assembly instructions can be found on the [Hackaday.io project page](https://hackaday.io/project/185211/instructions)!**
+<!--- I'll look at adding some reliable sources down the track for new ones - Sebyon --->
```
Comment (Phoenix1747) [inline hardware/README.md:59]: Some recommendations:

- https://www.ost-photonics.com/
- https://www.epic-crystal.com/

Both are Chinese manufacturers, but seem to provide good quality crystals and are pretty cheap too. Epic Crystal's scintillators are also sold by Gammaspectacular.

**Review by Phoenix1747 — COMMENTED** [review]
Nice, thanks for your contributions!

File: hardware/README.md (hunk 1 above)
Comment (sebyon) [reply hardware/README.md:10]: So with the TR, there is the MOQ from the supplier for 5000 units (the whole reel I assume), while the TR1 is jst the tape from the manufacturer. Likely the TR1 was purchased before the TR, or Mouser / Digikey just automatically bought both and just recuperate cost based on some back-end default profit margin based on the initial order. 

Can see this with the datasheet on page 15. 
[MICROC_SERIES_D-2315622.pdf](https://github.com/OpenGammaProject/Open-Gamma-Detector/files/11296267/MICROC_SERIES_D-2315622.pdf)

File: hardware/README.md (hunk 2 above)
Comment (sebyon) [reply hardware/README.md:59]: I think these are the two major Chinese new crystals that don't cost an arm and a leg from my knowledge.

File: hardware/README.md [hunk 3]
```diff
@@ -70,6 +70,27 @@ If the scintillator is too large, the output voltages will be very high and thus
...
 
+### Index-Matching Material
+
+You should apply some optical coupling compound between the SiPM and the crystal window to reduce reflections as good as possible (this way the best photon detection efficiency is achieved). There a multitude of options for optical couplants, and depending on your region some may be easier to aquire than others. As a general rule of thumb, silicone oil/grease can be used. If it contains polydimethylsiloxane, you're on the right track. Some options:
+
+<!---This is a good source https://www.logwell.com/tech/servtips/optical_coupling_grease.html - Sebyon --->
+
+| Description            | Brand  | Reflective Index | Viscosity (cPs)| Cost (USD) | Comment |
+|------------------------|--------|------------------|----------------|------------|---------|
+|[Index Matching Gel, 3 cc](https://www.thorlabs.com/thorproduct.cfm?partnumber=G608N3)|Thorlabs| 1.43 | 1,060,000 | $47.46 | High viscosity, works welll for fibre optic coupling. |
```
Comment (Phoenix1747) [inline hardware/README.md:81]: typo "works welll"

File: hardware/README.md (hunk 2 above)
Comment (Phoenix1747) [reply hardware/README.md:59]: Yup, I've had pretty good experiences with OST Photonics so far. I think they're also the cheapest actually. Do you want to add something to this section right now or do you want to wait and do some more research?

**Review by Phoenix1747 — CHANGES_REQUESTED** [review]
Typos, other than that LGTM.

File: hardware/README.md (hunk 3 above)
Comment (sebyon) [reply hardware/README.md]: I'll fix this in next pull request.

File: hardware/README.md [hunk 4]
```diff
@@ -70,6 +70,27 @@ If the scintillator is too large, the output voltages will be very high and thus
...
+|[Index-Matching Fluid](https://www.newport.com/f/index-matching-fluid) | Newport | 1.52 | 100 | $45.60 | Low viscosity may made setting the SiPM and crystal window difficult. However, the RI is the closer to the window properties. Use if you're experienced with assembly. |
```
[the remark this answers is not in the document]
Comment (sebyon) [reply hardware/README.md]: Fixed this typo in next pull.

**Review by Phoenix1747 — APPROVED** [review]
Great, thanks! 👍🏻

**Pull request merged**